### PR TITLE
document how to regenerate docs screenshots

### PR DESCRIPTION
**Pull request by sryza**
## Checklist

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -135,3 +135,22 @@ By default, we just display the object name. To override, use the `displayText`
...
+* A `path` to an image file that the screenshot should be stored in.
+* A `defs_file` that the script will run `dagit -f` or `dagit -w` on to load a set of Dagster definitions.
+* A `url` for the page to take the screenshot from.
+* An optional set of manual `steps`, which the person generating the screenshot is expected to carry out before the screenshot is taken.
+* A `vetted` boolean, which indicates whether the current screenshot in the repo was generated using the spec.
+
+Having written a screenshot spec, you can (re-)generate the screenshot by running the capture-screenshot script and providing the `path` value that you want to regenerate the screenshot for. E.g.
+
+```
+python docs/screenshot_capture/capture_screenshot.py concepts/dagit/runs-tab.png
```
Comment (gibsondan) [inline docs/README.md:153]: s/capture_screenshot.py/capture-screenshot.py

File: docs/README.md [hunk 2]
```diff
@@ -135,3 +135,22 @@ By default, we just display the object name. To override, use the `displayText`
 ### Navigation
 
 If you are adding a new page or want to update the navigation in the sidebar, update the `docs/content/_navigation.json` file.
+
+### Generating and Updating Screenshots
+
```
Comment (gibsondan) [inline docs/README.md:140]: also requires a 'pip install selenium'

Comment (gibsondan) [reply docs/README.md:140]: Other stuff I had to do:
download chromedriver from here https://chromedriver.chromium.org/downloads
Add it to /usr/local/bin
Run `xattr -d com.apple.quarantine /usr/local/bin/chromedriver` (gross)

**Review by gibsondan — APPROVED** [review]
used these instructions successfully, thanks! see suggestions inline

File: docs/README.md (hunk 2 above)
Comment (sryza) [reply docs/README.md:140]: Hmm I feel like how I did this was less gross, but now it's escaping me

**Pull request merged**